source/puppet/5.0/_puppet_toc.html

```diff
@@ -128,7 +128,7 @@ <h3 id="puppet-reference-manual">Puppet Platform reference manual</h3>
             * [HTTP client metrics]({{puppetserver}}/http_client_metrics.html)
         * [Tuning guide]({{puppetserver}}/tuning_guide.html)
         * [Applying metrics to improve performance]({{puppetserver}}/puppet_server_metrics_performance.html)
-        * [Scaling Puppet Server]({{puppetserver}}/scaling_multiple_masters.html)
+        * [Scaling Puppet Server]({{puppetserver}}/scaling_puppet_server.html)
```

Comment:
To confirm, this should link to https://docs.puppet.com/puppetserver/5.0/scaling_puppet_server.html, which I've confirmed works.